src/Workspaces/Remote/Core/IRemoteInitializationService.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.Remote;
+
+internal interface IRemoteInitializationService
+{
+    /// <summary>
+    /// Initializes values including <see cref="WorkspaceConfigurationOptions"/> for the process.
+    /// Called as soon as the remote process is created but can't guarantee that solution entities (projects, documents, syntax trees) have not been created beforehand.
```

Comment:
> can't guarantee that solution entities (projects, documents, syntax trees) have not been created beforehand

Is that possible, since any of the entities we would have created would have needed MEF to be created first? I think this comment you copied/pasted and its stale now.

Reply:
Will remove, it didn't make sense to me, but I don't have a lot of context in this area.